pkg/providers/v1/well_known_labels.go

```diff
@@ -20,4 +20,5 @@ const (
 	// LabelZoneID is a topology label that can be applied to any resource
 	// but will be initially applied to nodes.
 	LabelZoneID = "topology.k8s.aws/zone-id"
+	LabelNetworkNode = "topology.k8s.aws/network-node-"
```

Comment:
```suggestion
	LabelNetworkNode = "topology.k8s.aws/network-node-layer-"
```

IIUC, the info you're embedding into the tag is about the layer of the network node in the node set

Reply:
revised in next version